util/packaging/homebrew/chapel-main.rb

```diff
@@ -1,20 +1,17 @@
 class Chapel < Formula
   desc "Programming language for productive parallel computing at scale"
   homepage "https://chapel-lang.org/"
-  url "https://github.com/chapel-lang/chapel/releases/download/1.26/chapel-1.26.0.tar.gz"
-  sha256 "ba396b581f0a17f8da3f365a3f8b079b8d2e229a393fbd1756966b0019931ece"
-#  url "<URL of Chapel tarball>"
-#  sha256 "<sha256 sum for the associated tarball>"
+  url "https://github.com/chapel-lang/chapel/releases/download/1.27.0/chapel-1.27.0.tar.gz"
```

Comment:
I would think that we would want this to work with a `git` checkout of `main`, somehow. If that is future work then this is fine.

Reply:
Will need to look at that for testing and automation of the sha256 sum generation.   Captured as an issue...